### ISSUE TITLE
There is visually hidden live region content on the Date Picker documentation page that should not be visually hidden or in a live region.

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4559]
[Clarity Version]
3.1.1

[Test URL]
https://clarity.design/documentation/datepicker

[WCAG 2.1 Guideline]
1.3.2 Meaningful Sequence
(Level A)
https://www.w3.org/TR/WCAG21/#meaningful-sequence

[OS]
any

[User Agents/Assistive Technologies]
any

[Component]
Date Picker

[Description]
There is visually hidden live region content on the Date Picker documentation page that should not be visually hidden or in a live region.

See the relevant HTML/CSS below.

&lt;div id=&quot;clr-aria-live-element-clr-id-3&quot; class=&quot;clr-sr-only&quot; aria-atomic=&quot;true&quot; aria-live=&quot;polite&quot;&gt; It is recommended that you set the type to date as it increases readability of your code, helps with SEO, and keeps your HTML markup semantic. When the date picker is enabled, this type is overridden to text to disable the built-in date pickers that some desktop browsers provide. &lt;/div&gt;
&lt;div id=&quot;clr-aria-live-element-clr-id-7&quot; class=&quot;clr-sr-only&quot; aria-atomic=&quot;true&quot; aria-live=&quot;polite&quot;&gt; We recommend against overriding the automatically generated placeholder. Adding an incorrect placeholder will create confusion while entering the date in the input. &lt;/div&gt;
&lt;div id=&quot;clr-aria-live-element-clr-id-8&quot; class=&quot;clr-sr-only&quot; aria-atomic=&quot;true&quot; aria-live=&quot;polite&quot;&gt; The date picker uses the locale set for the application to ensure consistency of date inputs and outputs across the application. It is not recommended to override the locale identifier to use different locales for different date picker instances in the same application. &lt;/div&gt;
&lt;div id=&quot;clr-aria-live-element-clr-id-12&quot; class=&quot;clr-sr-only&quot; aria-atomic=&quot;true&quot; aria-live=&quot;polite&quot;&gt; Date picker emits null when an invalid date is entered after a valid date was set. &lt;/div&gt;
&lt;div id=&quot;clr-aria-live-element-clr-id-11&quot; class=&quot;clr-sr-only&quot; aria-atomic=&quot;true&quot; aria-live=&quot;polite&quot;&gt; We recommend that you either access the date as a string or as a javascript date object from the date picker. Using both at the same time might cause unexpected behavior. &lt;/div&gt;
&lt;div id=&quot;clr-aria-live-element-clr-id-1&quot; class=&quot;clr-sr-only&quot; aria-atomic=&quot;true&quot; aria-live=&quot;polite&quot;&gt; Clarity Date Picker is shown at screen width 768px and above. &lt;/div&gt;

[User Impact]
Blind screen reader users will hear a lot of content that sighted users cannot see. 

[Steps to Reproduce]
- Go to the Test URL
- Start a screen reader
- Refresh the page
- Observe that you hear the content mentioned in the description and that is not visible.

[Recommendation]
Present the content to all users. Remove the sr-only class, aria-live and aria-atomic.
